src/sections/community/styles.scss

```diff
@@ -240,3 +481,81 @@
 .spacer80px {
   height: 80px;
 }
+
+
+// Rainbow glow behind initiative sections NOT WORKING
```

Comment:
Let me know if this is worth pursuing or not

Reply:
I think it should be possible if you have a duplicate image underneath, then blur it. This way you don't need to pass a shadow color for the glow either